composer.json

```diff
@@ -244,6 +244,14 @@
         "drupal/social": "self.version"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
```

Comment:
This change shouldn't be part of the profile's composer.json. It should be part of "root" composer.json of the created project using "social" profile.